config/config.go

```diff
@@ -681,6 +681,23 @@ func loadConfig(reader io.Reader, config *Local) error {
 	return dec.Decode(config)
 }
 
+// DNSBootstrapArray returns an array of one or more DNS Bootstrap identifiers
+func (cfg Local) DNSBootstrapArray(networkID protocol.NetworkID) (bootstrapArray []string) {
+	dnsBootstrapString := cfg.DNSBootstrap(networkID)
```

Comment:
Why not just return strings.Split(bootstrapString, ";") ?

Reply:
It does not handle all cases, for example, if there is a leading or trailing semicolon or whitespace.  https://medium.com/@mlowicki/strings-fieldsfunc-vs-strings-split-96c667912f78

Reply:
you can always use ` strings.Split(bootstrapString, "; ")`, right ?
it's much more intuitive than the code below.
We don't have to support any user input here. The config file is assumed to be crafted correctly.

Reply:
and if it's a concern, consumer can trim and ignore empties while iterating (or we could do the same and return a sanitized array).  This approach is overly complicated.

Reply:
I started with Split() but updated to use the FieldsFunc() with white space removal so that the test cases would work.  I will revert to split and remove some of the test cases from config_test.TestLocal_DNSBootstrapArray().